### ISSUE TITLE
Cannot move a specific ticket to another category

### ISSUE THREAD

**Issue by not-an-aardvark** [waffleio/waffle.io#2441]
On [this project](https://waffle.io/porygonco/porybox), if I try to move issue #18 from "Backlog" to "In Progress", a red notice appears at the top of the screen, saying "Could not move card #18".

The following error is logged to the JS console:

```
angular.js:10103 TypeError: Converting circular structure to JSON
    at Object.stringify (native)
    at oa (https://d2mzqhzagbaems.cloudfront.net/vendor/angular-1.2.27.min.js:14:300)
    at Td.e.defaults.transformRequest (https://d2mzqhzagbaems.cloudfront.net/vendor/angular-1.2.27.min.js:69:382)
    at https://d2mzqhzagbaems.cloudfront.net/vendor/angular-1.2.27.min.js:69:22
    at r (https://d2mzqhzagbaems.cloudfront.net/vendor/angular-1.2.27.min.js:7:290)
    at vc (https://d2mzqhzagbaems.cloudfront.net/vendor/angular-1.2.27.min.js:69:4)
    at f (https://d2mzqhzagbaems.cloudfront.net/vendor/angular-1.2.27.min.js:71:242)
    at J (https://d2mzqhzagbaems.cloudfront.net/vendor/angular-1.2.27.min.js:101:96)
    at https://d2mzqhzagbaems.cloudfront.net/vendor/angular-1.2.27.min.js:102:259
    at h.$eval (https://d2mzqhzagbaems.cloudfront.net/vendor/angular-1.2.27.min.js:113:32)(anonymous function) @ angular.js:10103(anonymous function) @ angular.js:7390J @ angular.js:11662(anonymous function) @ angular.js:11745h.$eval @ angular.js:12788h.$digest @ angular.js:12600h.$apply @ angular.js:12892dragEnd @ ng-sortable-1.3.2.min.js:1o.event.dispatch @ jquery-2.1.0.min.js:3r.handle @ jquery-2.1.0.min.js:3nrWrapper @ porybox:39
angular.js:10103 TypeError: Cannot read property '_refreshed' of undefined
    at responseError (main.min.js:7)
    at w (angular.js:11668)
    at angular.js:11801
    at h.$eval (angular.js:12788)
    at h.$digest (angular.js:12600)
    at h.$apply (angular.js:12892)
    at HTMLDocument.dragEnd (ng-sortable-1.3.2.min.js:1)
    at HTMLDocument.o.event.dispatch (jquery-2.1.0.min.js:3)
    at HTMLDocument.r.handle (jquery-2.1.0.min.js:3)
    at HTMLDocument.nrWrapper (porybox:39)
```

I think this occurred because I edited a pull request to say "Fixes #18", and the card didn't move properly. However, the initial problem occurred a few days ago (and the ticket has been unmovable since then), so it's possible there are details I'm forgetting.

**Comment by homeyer** [comment]
Thanks @not-an-aardvark, we moved this into our current iteration to investigate.

**Comment by homeyer** [comment]
Maybe a good way to reproduce, just cause a failing test by sending the wrong request and you'll see this:

![image](https://cloud.githubusercontent.com/assets/84629/13753361/44307e18-e9d8-11e5-9d6c-f01e1a2327b1.png)

**Comment by homeyer** [comment]
@not-an-aardvark we shipped a fix, can you let me know if you still experience it?

**Comment by not-an-aardvark** [comment]
The ticket in question was merged on github, so I can't reproduce it exactly the same way as before. But it looks like I'm able to move the ticket around again, so I think it's fixed.

Thanks!

**Comment by kjdelisle** [comment]
@homeyer Unfortunately, I can reproduce this problem.

Originally, we had a ticket linked across a series of other tickets (meant to act as an epic), and then decided to invert that relationship by instead breaking those links and marking the epic with the individual items instead.

Example:
```
Epic issue - Do a bunch of stuff

closes #123
closes #124
closes #125

etc...
```

This was so the "epic" card would fit under all of the individual items that were a part of it.

Since then, attempting to move any of those items results in this error: 
![image](https://cloud.githubusercontent.com/assets/2053534/24592132/2184dad6-17df-11e7-9e35-cb3b5b75350e.png)

**Comment by BAMason** [comment]
When issues reference each other (issue A fixes B; B fixes A) Waffle now will not show either. This isn't necessarily by design, and doesn't solve the root issue, but is moderately better than JS errors :).

I'm adding this to our `Epic` milestone, since the end goal is to improve how we represent groups of related issues under a common parent.
